cfgov/tccp/jinja2/tccp/landing_page.html

```diff
@@ -102,27 +100,26 @@ <h2>
 
                     {{ form.location }}
 
-                    <div class="a-form-alert a-form-alert__error u-visually-hidden" id="location-required" role="alert">
+                    <div class="a-form-alert a-form-alert--error u-visually-hidden" id="location-required" role="alert">
                         {{ svg_icon("error-round") }}
-                        <span class="a-form-alert__text">
+                        <span class="a-form-alert--text">
```

Comment:
This looks wrong. A modifier should be together with another class it is modifying. Looks like it should be kept as `a-form-alert__text` (element of `a-form-alert`)